shared/src/main/scala/org/adridadou/openlaw/parser/template/variableTypes/ExternalCall.scala

```diff
@@ -21,6 +21,9 @@ import org.adridadou.openlaw.vm.OpenlawExecutionEngine
 object IntegratedServiceDefinition {
   val parser = new OpenlawTemplateLanguageParserService(Clock.systemUTC())
   val engine = new OpenlawExecutionEngine()
+  private val signatureDefinitionStr = "[[Input:Structure(signerEmail: Text; contractContentBase64: Text; contractTitle: Text)]] [[Output:Structure(signerEmail: Text; signature: Text; recordLink: Text)]]"
+
+  val Success(signatureDefinition) = IntegratedServiceDefinition(signatureDefinitionStr)
```

Comment:
I think this will throw an exception if the IntegratedServiceDefinition constructor fails.  Do we want to handle the error case more gracefully?

Reply:
We should clean that up at some point but right now I think we should leave it like that